packages/client/wallets/aa/src/utils/helpers.ts

```diff
@@ -18,8 +18,8 @@ export function transformBackwardsCompatibleChains<B extends BlockchainIncluding
     chain: B | BackwardsCompatibleChains
 ): B {
     switch (chain) {
-        case "mumbai":
-            return "polygon-mumbai" as B;
+        case "amoy":
+            return "polygon-amoy" as B;
```

Comment:
I don't think we need this, right?
We had this because previously we used `mumbai` but converted it later to `"polygon-mumbai"`

Basically I think we can remove this case entirely
```suggestion
```